NBXplorer.Tests/UnitTest1.cs

```diff
@@ -2663,6 +2663,21 @@ public void CanTrack()
 				tester.Notifications.WaitForBlocks(blockId);
 				var savedTx = tester.Client.GetTransaction(txId);
 				Assert.Equal(blockId[0], savedTx.BlockId);
+
+				var balanceBefore = tester.Client.GetBalance(pubkey);
+				Logs.Tester.LogInformation("Let's mine, and check that the balance is not updated until maturity occurs");
+				var blkid = tester.RPC.GenerateToAddress(1, tester.AddressOf(key, "0/0"))[0];
+				var blk = tester.RPC.GetBlock(blkid);
+				var minedTxId = blk.Transactions[0].GetHash();
+				tester.Notifications.WaitForTransaction(pubkey, minedTxId);
+				var balanceAfter = tester.Client.GetBalance(pubkey);
+				Assert.Equal(Money.Coins(50m), (Money)balanceAfter.Total - (Money)balanceAfter.Available);
+				var utxos = tester.Client.GetUTXOs(pubkey);
+				Assert.DoesNotContain(utxos.Confirmed.UTXOs, u => u.Outpoint.Hash == minedTxId);
+				Assert.DoesNotContain(utxos.Unconfirmed.UTXOs, u => u.Outpoint.Hash == minedTxId);
+				var transactions = tester.Client.GetTransactions(pubkey);
+				Assert.DoesNotContain(transactions.ConfirmedTransactions.Transactions, u => u.TransactionId == minedTxId);
+				Assert.Contains(transactions.ImmatureTransactions.Transactions, u => u.TransactionId == minedTxId);
```

Comment:
I think we could add another part to the test, that when the coinbase txn do get matured they are included in confirmed txn list.

Reply:
Done